cloud/openstack/os_server.py

```diff
@@ -360,13 +361,18 @@ def _create_server(module, cloud):
             metas[k] = v
         module.params['meta'] = metas
 
+    if type(module.params['security_groups']) is str:
+        module.params['security_groups'] = module.params['security_groups'].split(',')
```

Comment:
Also, just for future reference, it is better to use `isinstance` instead of running `type` on a variable and comparing it to a type.